addons/web_editor/static/src/js/common/grid_layout_utils.js

```diff
@@ -169,9 +169,9 @@ function _placeColumns(columnEls, rowSize, rowGap, columnSize, columnGap) {
         // Finding out if the images are alone in their column.
         let isImageColumn = _checkIfImageColumn(columnEl);
         const imageEl = columnEl.querySelector('img');
-        // Checking if the column has a background color to take that into
+        // Checking if the column has a background color or background image to take that into
         // account when computing its size and padding (to make it look good).
-        const hasBackgroundColor = columnEl.classList.contains("o_cc");
+        const hasBackgroundColor = columnEl.classList.contains("o_cc") || columnEl.classList.contains("oe_img_bg");
```

Comment:
Nitpick: this could be written as `columnEl.matches(".o_cc, .oe_img_bg")` too
But as you prefer :+1: